cadence/transactions/User/liquidate.cdc

```diff
@@ -13,9 +13,9 @@ import SwapRouter from 0x2f8af5ed05bbde0d
 // Repaied LIQUIDATEAMOUNT LIQUIDATETOKEN on your behalf
 //
 transaction(repayPoolAddr: Address, seizePoolAddr: Address, amountLiquidate: UFix64, borrower: Address) {
-    prepare(signer: AuthAccount) {
-        let poolRepayRef = getAccount(repayPoolAddr).getCapability<&{LendingInterfaces.PoolPublic}>(LendingConfig.PoolPublicPublicPath).borrow()!
-        let poolSeizeRef = getAccount(seizePoolAddr).getCapability<&{LendingInterfaces.PoolPublic}>(LendingConfig.PoolPublicPublicPath).borrow()!
+    prepare(signer: auth(Storage, Capabilities) &Account) {
```

Comment:
In general, I'd recommend using minimum required account entitlements, in particular with respect to `Capabilities`. Using `Capabilities` enables the issuance of `Account` Capabilities which is highly sensitive. There's a lot going on in the transaction, so maybe `StorageCapabilities` would cover all cases while still being a bit more fine grained.

Reply:
Done